packages/js-dapi-client/lib/transport/DAPIStream.js

```diff
@@ -0,0 +1,129 @@
+const EventEmitter = require('events');
+const GrpcErrorCodes = require('@dashevo/grpc-common/lib/server/error/GrpcErrorCodes');
+
+const defaultOptions = {
+  reconnectTimeoutDelay: 50000,
```

Comment:
Let's add a TODO to increase it when we get rid of timeout from Nginx

Reply:
Done